src/main/groovy/io/seqera/wave/controller/CondaController.groovy

```diff
@@ -0,0 +1,61 @@
+/*
+ *  Wave, containers provisioning service
+ *  Copyright (c) 2023-2024, Seqera Labs
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <https://www.gnu.org/licenses/>.
+ */
+
+package io.seqera.wave.controller
+
+
+import groovy.transform.CompileStatic
+import groovy.util.logging.Slf4j
+import io.micronaut.core.annotation.Nullable
+import io.micronaut.http.HttpResponse
+import io.micronaut.http.annotation.Controller
+import io.micronaut.http.annotation.Get
+import io.micronaut.scheduling.TaskExecutors
+import io.micronaut.scheduling.annotation.ExecuteOn
+import io.seqera.wave.service.conda.CondaFetcherService
+import io.seqera.wave.service.persistence.CondaPackageRecord
+import io.seqera.wave.service.persistence.PersistenceService
+import jakarta.inject.Inject
+/**
+ *
+ * @author Paolo Di Tommaso <paolo.ditommaso@gmail.com>
+ */
+@Slf4j
+@CompileStatic
+@Controller("/")
+@ExecuteOn(TaskExecutors.IO)
+class CondaController {
+
+    @Inject
+    PersistenceService persistenceService
+
+    @Inject
+    CondaFetcherService condaService
+
+    @Get('/v1alpha1/conda{?search}')
+    HttpResponse<List<CondaPackageRecord>> list(@Nullable String search) {
+        return HttpResponse.ok(persistenceService.findCondaPackage(search))
+    }
+
+    @Get('/v1alpha1/conda/update')
```

Comment:
```suggestion
    @Get('/v1alpha1/packages/conda/refresh')
```

Reply:
@pditommaso I think we should not expose this because it is being called by Corn Job and exposing it might leads to DOS attack

Reply:
or it should be authenticated

Reply:
let's keep for now only for testing

Reply:
done